Constellation/Node/Api.hs

```diff
@@ -113,6 +139,38 @@ data Whitelist = Whitelist
     , wlIPv6 :: Set HostAddress6
     } deriving Show
 
+decodeSendRaw :: BL.ByteString -> RequestHeaders -> Either String Send
+decodeSendRaw b h = case getHeaders [hContentLength, hFrom, hTo] h of
+    Right headers -> Right Send
+                    { sreqPayload = decodePayload (hmap ! hContentLength) b
+                    , sreqFrom    = mustDecodeB64PublicKey $ hmap ! hFrom
```

Comment:
They'll still be CI ByteStrings (HeaderName) if given literally (at least with OverloadedStrings.)

Hiding their actual values away causes reader to have to figure out where hFrom and hTo are declared, which doesn't seem worth it given they're not used in more than one place (+test.)

If you want to keep them but move them to Node.Api so they're in one place, that works.